frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -60,20 +64,18 @@ const menuActions = [
 ];
 
 export const PVCStatus = ({ pvc }) => {
-  const pvcStatusExtensions = useExtensions(isPVCStatus);
+  const [pvcStatusExtensions] = useExtensions(isPVCStatus);
```

Comment:
`useExtensions` hook result is the extensions array itself, so no need for `[ .. ]` destructuring.

`useResolvedExtensions` hook result, on the other hand, is a tuple `[resolvedExtensions, resolved, errors]` so that's a good case for destructuring.

Reply:
```suggestion
  const pvcStatusExtensions = useExtensions(isPVCStatus);
```